packages/arb-token-bridge-ui/src/components/TransferPanel/CctpTabContent.tsx

```diff
@@ -0,0 +1,45 @@
+import { PropsWithChildren } from 'react'
+import { CCTP_DOCUMENTATION } from '../../constants'
+import { useCCTPIsBlocked } from '../../hooks/CCTP/useCCTPIsBlocked'
+import { ExternalLink } from '../common/ExternalLink'
+
+export const CctpTabContent = ({
+  toNetworkName,
+  children
+}: PropsWithChildren<{
+  toNetworkName: string
+}>) => {
+  const { data: isCctpBlocked, isLoading: isLoadingIsCctpBlocked } =
+    useCCTPIsBlocked()
+
+  if (isLoadingIsCctpBlocked || isCctpBlocked) {
+    return (
+      <>
+        Access to Circle&apos;s bridge is restricted in certain jurisdictions.
+        For more details, please consult Circle&apos;s{' '}
+        <ExternalLink
+          className="arb-hover text-blue-link underline"
+          href={CCTP_DOCUMENTATION}
```

Comment:
we're going to update this link when there's a more specific page right?

Reply:
If there is, then yes we will update